packages/help-center/src/hooks/use-user-is-admin-atomic.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-restricted-imports */
+
+import { HelpCenterSelect } from '@automattic/data-stores';
+import { useSelect } from '@wordpress/data';
+import useUsersQuery from 'calypso/data/users/use-users-query';
+import { useSelector } from 'calypso/state';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
+import { HELP_CENTER_STORE } from '../stores';
+
+export default function useUserIsAtomicAdmin() {
+	const currentSite = useSelect( ( select ) => {
+		const helpCenterSelect: HelpCenterSelect = select( HELP_CENTER_STORE );
+		return helpCenterSelect.getSite();
+	}, [] );
+	const isAtomicSite = currentSite?.is_wpcom_atomic;
+	const currentUser = useSelector( getCurrentUser );
```

Comment:
In the context of the admin bar, for an atomic site we don't have this info.
The currentUser will be empty